### DISCUSSION TITLE
AA for Q5

### DISCUSSION THREAD

**Discussion by LukeQ5** [Mr-MIBonk/M.I.B._More-Incredible-Bash#165] [Q&A] [answered]
Hi, i have a Q5 with MIB2 FW MHI2Q_ER_AUG22_P4212 MU 1177. I want activate AC & AA and I need some help, i have usb port only charge.

1. Do I need to buy a USB port for CarPlay?
2. Do I need install M.I.B.?
3. Do I need run Backup?
4. Do I need to install the AIO patch to activate AA or can I activate directly from other menu?
5. If I buy the kit 2 USB ports (8W0.035.736) can I activate both?

Any help is appreciated
Thanks in advance
Luca

**Comment by Mr-MIBonk** [comment, marked as answer]
All you need is to read the wiki!

**Reply by LukeQ5** [reply]
Tks

**Comment by harman-f** [comment]
Mibwiki.one

**Reply by LukeQ5** [reply]
Ok tks
